azkaban-exec-server/src/main/java/azkaban/execapp/FlowRunner.java

```diff
@@ -768,46 +772,46 @@ private Props loadJobProps(final ExecutableNode node) throws IOException {
     return props;
   }
 
-  private Props loadFlowPropsFromYamlFile() {
-    final File flowFile = getFlowFile();
-    final Props flowProps = FlowLoaderUtils.getPropsFromYamlFile(this.flow.getId(), flowFile);
-    if (flowFile != null && flowFile.exists()) {
-      flowFile.delete();
+  private Props loadPropsFromYamlFileUtil(final String path) {
```

Comment:
What cause you to change the method name? To me, `loadPropsFromYaml` sounds good enough.

Reply:
I'm ok with either name.